app/models/permitted_params.rb

```diff
@@ -1,4 +1,4 @@
-class PermittedParams < Struct.new(:params, :user)
+class PermittedParams < Struct.new(:params)
```

Comment:
Heh, I didn't know you could do `PermittedParams < Struct.new`; I'd always seen it as `PermittedParams = Struct.new`, cool.